examples/log-extension/package.json

```diff
@@ -48,7 +48,8 @@
     "index.d.ts",
     "dist*/src",
     "dist*/index*",
-    "src"
+    "src",
+    "test"
```

Comment:
This is a good point!

Conceptually, we have two kinds of `npm publish` for our example projects:

 - We need to publish all files needed to develop the example because we are using `npm` to obtain the example project.
 - To make it easy for users to convert the example project into a real one, we need to support "real" publishing of runtime artifacts only (`index.*`, `dist*`, but not `test`).

One option that comes to my mind:

- Instead of removing "files" section, rename the section to something else, e.g. "production-files". This way the initial `npm publish` from our monorepo packages all files.
- Modify the implementation of `lb4 example` to rename `production-files` back to `files`. This way `npm publish` from the cloned example packages only the relevant runtime/production files.

@strongloop/lb-next-dev thoughts?